apicast/src/configuration_loader.lua

```diff
@@ -128,7 +128,11 @@ end
 function boot.init_worker(configuration)
   local interval = ttl() or 0
 
-  configuration.keycloak = keycloak.load_configuration()
+  local keycloak_config = _M.run_external_command("keycloak")
```

Comment:
And it is already loaded. Init_worker is executed right after. So it should be removed.